src/Microsoft.ML.AutoML/Utils/Logger.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+
+namespace Microsoft.ML.AutoML
+{
+    internal class AutoMLLogger
```

Comment:
(minor, doesn't need to be solved for this PR) -

I wonder if this object has enough value over just using the `IChannel` itself.

Reply:
AFAIR this used to be a bit more elaborate but got trimmed down since, removed it.  I don't like how it hides MessageSensitivity and defaults it to None.  Thought about if it's reasonable to rename logger -> channel, decided to stick with logger since that's how it is really used.